### DISCUSSION TITLE
Comparazione sensori

### DISCUSSION THREAD

**Discussion by evirci** [indomus/forum#2344] [GENERALE - Discussioni generiche]
Buonasera a tutti!
Qualche anima pia sarebbe in grado di spiegarmi come ricavare il valore massimo di un sensore di temperatura salvando il dato in un altro sensore?
Nella mia testa penso che bisogna confrontare i due sensori, quando la temperatura rilevata dal sensore 1 è maggiore della temperatura salvata nel sensore 2, aggiorno il dato sul sensore 2.

Secondo voi è fattibile?
Grazie mille

**Comment by snakuzzo** [comment]
Non ho capito qual è lo scopo di quello che stai facendo e di quello che hai chiesto.
Spiega il caso d'uso...

Ad ogni modo, il valore massimo di una lista di valori numerici si ottiene con il filter `max`

![image](https://user-images.githubusercontent.com/7205371/219113144-7c07b213-479a-46a4-8f7d-d2cbcce12211.png)

**Reply by evirci** [reply]
cerco di spiegarmi meglio...
Siccome ho la temperatura del processore altina, mi interesserebbe avere un sensore che mi mostri la temperatura massima raggiunta dalla CPU nell'arco di una settimana...La stessa funzione la userei per la temperatura del frigo.

**Reply by snakuzzo** [reply]
Secondo me stai sbagliando approccio con i sensori che dici tu.
Credo tu debba usare -> https://www.home-assistant.io/integrations/statistics/

**Reply by evirci** [reply]
ok, ho creato correttamente il sensore, ma come faccio a prendere i dati dal sensore intime?
https://pastebin.com/YgWmQ1BD

**Comment by indomus** [comment]
> Buonasera a tutti! Qualche anima pia sarebbe in grado di spiegarmi come ricavare il valore massimo di un sensore di temperatura salvando il dato in un altro sensore? Nella mia testa penso che bisogna confrontare i due sensori, quando la temperatura rilevata dal sensore 1 è maggiore della temperatura salvata nel sensore 2, aggiorno il dato sul sensore 2.
> 
> Secondo voi è fattibile? Grazie mille

tu parli sostanzialmente di sensori offset.
prima di creare sensori, [leggi questo](https://indomus.it/progetti/calibrare-i-sensori-ambientali-su-home-assistant/).

**Reply by evirci** [reply]
sarò tonto io, ma non riesco a capire come potrebbe essermi d'aiuto...
Questa funzione invece sembra essere fatta ad hoc, ma non sembra funzionare... forse è perchè vuole più entità e non una sola

[https://www.home-assistant.io/integrations/min_max/](url)

**Reply by evirci** [reply]
ho provato a creare anche più entità in ingresso, nulla... segna lo stesso valore del sensore "padre"
